flink-connector-influxdb2/src/main/java/org/apache/flink/streaming/connectors/influxdb/common/DataPoint.java

```diff
@@ -25,8 +25,12 @@
 import javax.annotation.Nullable;
 import lombok.Getter;
 
-/** InfluxDB data points */
```

Comment:
General Code Notes:
- parseTimestamp -> null-Case mit return nach oben ziehen
- commit -> !this.writeCheckpoint-Case mit return nach oben ziehen
- writeCheckpointDataPoint -> wirklich kein Error-Log im Exception Case notwendig? falls nein: Kommentar hinzufügen
- `InfluxDBWriter#write` has `e.printStackTrace()` instead of proper logging
- `InfluxDBSource` has TODO example
- `InfluxDBSourceEnumStateSerializer` needs a comment why no serialization is required and the class is a stub
- `InfluxDBSplitSerializer` needs a comment why `flip()` is needed
- root `pom.xml` has formatting & unrelated changes → may want to revert them